nall/image.hpp

```diff
@@ -157,6 +157,30 @@ struct image {
   channel _blue {255u <<  0, 8,  0};
 };
 
+struct multiFactorImage : public image {
+    using image::image;
+
+    multiFactorImage(const multiFactorImage& source);
+    multiFactorImage(multiFactorImage&& source);
+    multiFactorImage(const image& lowDPI, const image& highDPI);
```

Comment:
If this is going to be a "multifactor" image, instead of just a "bifactor" image, shouldn't the API support an arbitrary number of factors?

Reply:
Yes and no, I used multi-factor because some platforms might benefit from "mid DPI" (x1.5) or "very high DPI" (x3) images, but since I'm not very familiar with GUI development on these I didn't want to commit to a specific API. I went with something that is currently limited to two factors but can be later be extended with more methods as needed by other frontends.